src/cmd/check.go

```diff
@@ -92,31 +92,72 @@ type CheckCreateType struct {
 	Spec map[string]interface{}
 }
 
-func (self *CheckCreateType) resolveAliases() {
+func (self *CheckCreateType) resolveCategoryAliases(client *opslevel.Client) (error) {
```

Comment:
Took a stab at refactoring these based on what I had to do for the integration one